mediaqueries/Overview.bs

```diff
@@ -491,7 +491,7 @@ Evaluating Media Features in a Boolean Context</h4>
 	If the feature would be true for any value
 	<em>other than</em> the number ''0'',
 	a <<dimension>> with the value ''0'',
-	or the keyword ''none'',
+	or the keywords ''none'' and ''default'',
```

Comment:
Please change `default` to `no-preference` per [discussion](https://github.com/w3c/csswg-drafts/issues/442#issuecomment-266637739)